Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -345,10 +345,12 @@ void RenderBlockFlow::adjustIntrinsicLogicalWidthsForColumns(LayoutUnit& minLogi
 
 void RenderBlockFlow::computeIntrinsicLogicalWidths(LayoutUnit& minLogicalWidth, LayoutUnit& maxLogicalWidth) const
 {
+    bool needAdjustIntrinsicLogicalWidthsForColumns = true;
     if (shouldApplySizeContainment()) {
```

Comment:
@Loirooriol https://github.com/WebKit/WebKit/pull/10797 would change this line to `shouldApplySizeOrInlineSizeContainment`. Others are not changed.